cpp/test/core/apply_tests.cpp

```diff
@@ -190,6 +413,14 @@ void ApplyTests(const char* name)
     AddTestCase<TEST_ID(N), AllTests, Reader, Writer>(suite, "Use generated *_apply.cpp");
 }
 
+template <uint16_t N>
```

Comment:
minor: `std`?

Reply:
Same, trying to align with the existing style in the file at least.